db/schema.rb

```diff
@@ -0,0 +1,127 @@
+    t.integer "contribution_type", null: false
+    t.string "relationship", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["memorial_id"], name: "index_contributions_on_memorial_id"
+    t.index ["user_id"], name: "index_contributions_on_user_id"
+  end
+
+  create_table "flowers", charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+    t.bigint "contribution_id", null: false
+    t.integer "type", default: 0, null: false
+    t.string "message"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["contribution_id"], name: "index_flowers_on_contribution_id"
+  end
+
+  create_table "memorials", charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+    t.string "name", null: false
+    t.integer "gender"
+    t.string "cause_of_death"
+    t.bigint "user_id", null: false
+    t.string "background_song"
+    t.string "avatar"
+    t.string "relationship", null: false
+    t.integer "privacy_type", default: 0, null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["user_id"], name: "index_memorials_on_user_id"
+  end
+
+  create_table "placetimes", charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+    t.bigint "memorial_id", null: false
+    t.string "location"
+    t.datetime "date"
+    t.boolean "is_born"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["memorial_id"], name: "index_placetimes_on_memorial_id"
+  end
+
+  create_table "stories", charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+    t.bigint "contribution_id", null: false
+    t.string "image"
+    t.string "content", null: false
+    t.bigint "rep_story_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["contribution_id"], name: "index_stories_on_contribution_id"
+    t.index ["rep_story_id"], name: "index_stories_on_rep_story_id"
+  end
+
+  create_table "tributes", charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+    t.bigint "contribution_id", null: false
+    t.string "eulogy", null: false
+    t.bigint "rep_tribute_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["contribution_id"], name: "index_tributes_on_contribution_id"
+    t.index ["rep_tribute_id"], name: "index_tributes_on_rep_tribute_id"
+  end
+
+  create_table "users", charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+    t.string "name", null: false
+    t.integer "gender"
+    t.string "email", null: false
+    t.string "avatar"
+    t.integer "role_id", default: 0, null: false
```

Comment:
```suggestion
    t.integer "role", default: 0, null: false
```
ri hì, thêm cái `_id` chị nhầm là mapping với table á